core/src/main/java/org/arquillian/smart/testing/Configuration.java

```diff
@@ -45,6 +45,10 @@ public boolean isModeSet() {
         return this.mode != null;
     }
 
+    public boolean isSkipTests() {
+        return Boolean.valueOf(System.getProperty("maven.test.skip", "false"));
```

Comment:
also you can set it by using `skipTests`property. If it is set then using `-DskipTests` then tests are also skipped.